test/Plugin/Show/ShowFullOrOpenTables/ShowFullOrOpenTablesPayloadTest.php

```diff
@@ -152,8 +157,8 @@ public function testSQLQueryParsingFailedOnWrongSyntax(): void {
    *  path:string
    * }
    */
-	protected static function buildSQLQuery(array $args): array {
-		$query = 'SHOW FULL TABLES';
+	protected static function buildSQLQuery(array $args, string $queryType): array {
+		$query = "SHOW $queryType TABLES";
```

Comment:
I suggest use {$queryType} for better visibility
```suggestion
		$query = "SHOW {$queryType} TABLES";
```

Reply:
Ok